### DISCUSSION TITLE
Lps max number of blocks (16)

### DISCUSSION THREAD

**Discussion by marlos1494** [bitcraze/discussions#486] [Q&A]
i am new with crazyflie and i don't know so much about python . I am using LPS a configuration of 8 anchors and i did a scrip that print the position, i will use the position after for implement a control. when he commanded to call the position and print i get the next error. I'm not sure what I'm doing wrong. I  updated firmware of crazy and crazyradio but this not help. I hope can help me.

This is the error.

> Error no LogEntry to handle id=3
Error no LogEntry to handle id=3
Waiting for estimator to find position...
0 0 0
0 0 0
0 0 0
0 0 0
0 0 0
0 0 0
0 0 0
0 0 0
0 0 0
0 0 0
0 0 0
1.9211047887802124 -1.5065696239471436 0.9987784624099731
1.9211047887802124 -1.5065696239471436 0.9987784624099731
1.9277292490005493 -1.5074166059494019 0.9971343278884888
1.9277292490005493 -1.5074166059494019 0.9971343278884888
1.926999807357788 -1.5073847770690918 0.9981831908226013
1.926999807357788 -1.5073847770690918 0.9981831908226013
1.925742268562317 -1.5072834491729736 0.9998694658279419
1.925742268562317 -1.5072834491729736 0.9998694658279419
1.9210259914398193 -1.5068992376327515 1.003173828125
1.9210259914398193 -1.5068992376327515 1.003173828125
1.922640323638916 -1.5004549026489258 0.9975059032440186
Traceback (most recent call last):
  File "/home/labnav/crazyflie-lib-python/examples/med_position.py", line 230, in <module>
    act=take_off(scf,t)
  File "/home/labnav/crazyflie-lib-python/examples/med_position.py", line 158, in take_off
    start_position_printing(scf)
  File "/home/labnav/crazyflie-lib-python/examples/med_position.py", line 139, in start_position_printing
    log_conf.start()
  File "/home/labnav/crazyflie-lib-python/cflib/crazyflie/log.py", line 296, in start
    self.create()
  File "/home/labnav/crazyflie-lib-python/cflib/crazyflie/log.py", line 276, in create
    raise AttributeError(
AttributeError: Configuration has max number of blocks (16)

**Comment by knmcguire** [comment]
Hi! This is not an LPS problem but a logging problem. The error are indicated that you have hit the maximum of log block configurations, which might indicate that your script is initializing a new log-block within the loop for the same values.

Could you share a minimal example of your code or tell us on which example the code has been based on?
